doc/classes/Viewport.xml

```diff
@@ -149,6 +159,14 @@
 			<param index="0" name="event" type="InputEvent" />
 			<param index="1" name="in_local_coords" type="bool" default="false" />
 			<description>
+				Triggers the given [InputEvent] in this [Viewport]. This can be used to pass input events between viewports, or to locally apply inputs that were sent over the network or saved to a file.
+				If [param in_local_coords] is set to [code]true[/code], the given [param event] will be transformed to the local coordinate space of the [Viewport].
+				While this method serves a similar purpose as [method Input.parse_input_event], it does not remap the specified [param event] based on project settings like [member ProjectSettings.input_devices/pointing/emulate_touch_from_mouse].
+				Calling this method will propagate calls to child nodes for following methods in the given order:
+				- [method Node._shortcut_input]
+				- [method Node._unhandled_input]
+				- [method Node._unhandled_key_input]
+				If an earlier method marks the input as handled via [method set_input_as_handled], any later method in this list will not be called.
```

Comment:
Here would be the following additional information useful:
"If none of the methods handle the event and [member physics_object_picking] is [code]true[/code], the event is used for physics object picking."